assembl/static2/js/app/components/debate/brightMirror/sideComment/sideCommentBox.jsx

```diff
@@ -0,0 +1,333 @@
+// @flow
```

Comment:
this file is a ~ copy of harvestingBox.jsx. I think we have to do a refactoring

Reply:
not exactly but it can be done later if we wish to apply the changes I did for this module to the harvesting functionality